app/models/candidate_interface/sign_up_form.rb

```diff
@@ -12,6 +12,7 @@ def initialize(params = {})
       @email_address = params[:email_address]
       @accept_ts_and_cs = params[:accept_ts_and_cs]
       @candidate = Candidate.for_email @email_address
+      @course_from_find_id = params[:course_from_find_id]
```

Comment:
Hmm, does this work? It looks like we're passing in `course_id` instead of `course_from_find_id` from the controller.

Reply:
Ah, you already fixed it https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1165/commits/b5ea07666d50982c990dd208900e80a7a0500e36 👍